--- configure	2011-07-05 15:34:05.991022938 -0400
+++ configure	2011-07-05 15:34:29.599022943 -0400
@@ -5761,12 +5761,7 @@
 
 if test "x${enable_deprecated_flags}" = "xyes"; then
 
-  list="-DG_DISABLE_DEPRECATED
-		-DPANGO_DISABLE_DEPRECATED
-		-DGDK_DISABLE_DEPRECATED
-		-DGDK_PIXBUF_DISABLE_DEPRECATED
-		-DGTK_DISABLE_DEPRECATED
-		-DGNOME_DISABLE_DEPRECATED"
+  list=""
   flags_supported=""
   flags_unsupported=""
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for supported compiler flags" >&5

